walkthroughs/1-ionic-showcase/walkthrough-alternative.adoc

```diff
@@ -0,0 +1,247 @@
+:showcase-app-link: https://github.com/aerogear/ionic-showcase
```

Comment:
So this will be the lightweight showcase server and frontend focused walkthrough that does not involve MDC and takes a very little amount of time.